Authentication/AuthenticationBootstrapper.swift

```diff
@@ -162,7 +162,7 @@ public class AuthenticationBootstrapper<User: UserType, SessionService: SessionS
     }
     
     public func createSignupView() -> SignupViewType {
-        let view = SignupView.loadFromNib()
+        let view = SignupView.loadFromNib()!
```

Comment:
I don't think this method should return an optional since a view must be provided for the authentication process.
About the exception, we could, but I don't think it's necessary since this view comes with the framework, it will never fail.